src/JsonSchema/Enum.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace JsonSchema;
+
+class Enum extends \MabeEnum\Enum
```

Comment:
If this isn't intended to be extended-only (and might be used directly for something), then it doesn't need to be abstract, and probably shouldn't be. I was suggesting that based on an invalid assumption that this would always be extended for errors.